packages/block/README.md

```diff
@@ -57,6 +57,38 @@ try {
 
 [Documentation](./docs/README.md)
 
+# CONSENSUS TYPES
+
+The block library supports the creation as well as format and consensus validation of PoW `ethash` and PoA `clique` blocks.
+
+## Ethash/PoW
+
+TODO
```

Comment:
a note/reminder we should include reference to the BlockOption `calcDifficultyFromHeader`

Reply:
Just had another look: `calcDifficultyFromHeader` has already been added to `v3.0.0`. 😄